howto/install-appliance/google-cloud.md

```diff
@@ -0,0 +1,107 @@
+
+Configure the boot disk by selecting **Boot disk > CHANGE** to choose the operating system and boot disk size.
+
+![Configure boot disk|690x440](https://assets.ubuntu.com/v1/d85bed8c-boot-disk-config.png)
+
+Select the operating system. Google Cloud has two options - Ubuntu and Ubuntu Pro. If you have an Ubuntu Pro subscription already, you can choose Ubuntu and [attach your subscription](https://discourse.ubuntu.com/t/22681#attach-ubuntu-pro) manually. If you don’t, you can choose the Ubuntu Pro option which will include a Ubuntu Pro subscription through Google Cloud. 
+
+[note type="information" status="Note"]Remember that choosing the Ubuntu Pro option will have an impact on your pricing. The **Pricing summary** section provides the resource details and the associated costs.[/note]
+
+Select the latest Ubuntu image for the architecture that you want to use. For example, Ubuntu 22.04 LTS Pro Server for Arm64.
+
+Define your preferred boot disk size. We recommend allotting at least 100 GB of disk size.
+
+For Anbox Cloud Appliance, adding an additional disk is recommended for efficient container storage. Under **Advanced Options > Disks**, Select **ADD NEW DISK**. Add a disk name and choose the disk size as 100 GB. For the other fields, you can choose your preferred value or proceed with the default values. 
+
+![Add new disk|690x440](https://assets.ubuntu.com/v1/29645ed9-add-new-disk.png)
+
+### 4. Configure firewall mapping
+
+There are two ways to map the firewall rules to the virtual machine in Google Cloud:
+
+**Google service account** - If you want to use a service account to map your firewall rules to the virtual machine, you can use the default service account with default access scopes under the **Identity and API access** section. This service account will be used when creating a firewall rule.
+
+**Specified target tags** - If you want to use specified target tags to map your firewall rules to the virtual machine, assign a unique network tag under **Advanced options > Networking > Network tags**. This network tag will later be used while creating a firewall rule.
+
+### 5. Launch the virtual machine
+
+Select **CREATE** to create the virtual machine on Google Cloud. For additional information on creating a virtual machine in Google Cloud, see [Google Cloud documentation](https://cloud.google.com/compute/docs/instances/create-start-instance).
+
+## Firewall setup
+
+Before installing Anbox Cloud on the virtual machine, you should set up a firewall to control the incoming and outgoing traffic to your virtual machine. From the navigation menu, select **VPC network > Firewall**. Select **CREATE FIREWALL RULE**. 
+
+You can create a firewall rule based on your networking requirements following the [Google documentation](https://cloud.google.com/firewall/docs/using-firewalls). The following is an example firewall rule to enable access to the Anbox Cloud Appliance dashboard once the appliance is installed on the virtual machine. 
+
+````
```

Comment:
Shouldn't this be three backticks instead of four?

Reply:
Both work fine in VS code but am not sure how it will render in discourse, I am still getting familiar with some markdown syntax :) Thanks, I will correct it!